tests/_files/test_animation.py

```diff
@@ -191,6 +194,19 @@ async def make_assertion(url, request_data: RequestData, *args, **kwargs):
         monkeypatch.setattr(bot.request, "post", make_assertion)
         assert await bot.send_animation(animation=animation, chat_id=chat_id)
 
+    @pytest.mark.parametrize("bot_class", ["Bot", "ExtBot"])
+    async def test_send_animation_thumb_deprecation_warning(
+        self, recwarn, monkeypatch, bot_class, bot, raw_bot, chat_id, animation
+    ):
+        async def make_assertion(url, request_data: RequestData, *args, **kwargs):
+            return True
+
+        bot = raw_bot if bot_class == "Bot" else bot
```

Comment:
```suggestion
        bot = request.getfixturevalue("raw_bot/bot")
```
this would be way to dynamically load a fixture, just for reference. I'm okay with that approach also